networks/rpc/websocket.go

```diff
@@ -133,15 +133,29 @@ func (srv *Server) FastWebsocketHandler(ctx *fasthttp.RequestCtx) {
 // NewWSServer creates a new websocket RPC server around an API provider.
 //
 // Deprecated: use Server.WebsocketHandler
-func NewWSServer(allowedOrigins []string, srv *Server) *http.Server {
-	return &http.Server{Handler: srv.WebsocketHandler(allowedOrigins)}
+func NewWSServer(allowedOrigins []string, timeouts HTTPTimeouts, srv *Server) *http.Server {
+	timeouts = sanitizeTimeouts(timeouts)
+	return &http.Server{
+		Handler:      srv.WebsocketHandler(allowedOrigins),
+		ReadTimeout:  timeouts.ReadTimeout,
+		WriteTimeout: timeouts.WriteTimeout,
+		IdleTimeout:  timeouts.IdleTimeout,
+	}
 }
 
-func NewFastWSServer(allowedOrigins []string, srv *Server) *fasthttp.Server {
+func NewFastWSServer(allowedOrigins []string, timeouts HTTPTimeouts, srv *Server) *fasthttp.Server {
+	timeouts = sanitizeTimeouts(timeouts)
 	upgrader.CheckOrigin = wsFastHandshakeValidator(allowedOrigins)
 
 	// TODO-Klaytn concurreny default (256 * 1024), goroutine limit (8192)
-	return &fasthttp.Server{Concurrency: concurrencyLimit, MaxRequestBodySize: common.MaxRequestContentLength, Handler: srv.FastWebsocketHandler}
+	return &fasthttp.Server{
+		Concurrency:        concurrencyLimit,
+		MaxRequestBodySize: common.MaxRequestContentLength,
+		Handler:            srv.FastWebsocketHandler,
+		ReadTimeout:        timeouts.ReadTimeout,
+		WriteTimeout:       timeouts.WriteTimeout,
+		IdleTimeout:        timeouts.IdleTimeout,
+	}
```

Comment:
As you know, the added configurations are not applied to the websocket server. Could you leave comments for that if you want to add unused configurations?

Reply:
Right. I will add some comments.

Reply:
I acturally removed timeouts for WSServer. Please take a look.